storage/replica_test.go

```diff
@@ -798,7 +805,10 @@ func TestRangeNoGossipFromNonLeader(t *testing.T) {
 
 	// Make sure the information for db1 is not gossiped.
 	tc.rng.maybeGossipSystemConfig()
-	cfg, err := tc.gossip.GetSystemConfig()
+	// Fetch the raw gossip info, GetSystemConfig is based on
```

Comment:
Because we're specifically checking that the info was _not_ gossiped. I can't wait for a callback because there won't be one, so I need to check the raw value.